patches/tModLoader/Terraria/ModLoader/TileLoader.cs

```diff
@@ -730,13 +730,17 @@ public static void RegisterConversion(int tileType, int conversionType, ConvertT
 	/// <param name="conversionType">The conversion type for which the tile should use this conversion.</param>
 	/// <param name="toType">The tile type that this conversion should convert the tile to.</param>
 	/// <param name="purification">If true, automatically registers purification conversions from toType to tileType as well.</param>
-	public static void RegisterSimpleConversion(int tileType, int conversionType, int toType, bool purification = true)
+	/// <param name="fallback">If true, automatically registers tileType as fallback tile of toType.</param>
+	public static void RegisterSimpleConversion(int tileType, int conversionType, int toType, bool purification = true, bool fallback = true)
```

Comment:
I don't feel good about the ABI breakage of adding a new parameter, and I figured out a better solution in Discord.